tinyms/model/deepfm/deepfm_train_model.py

```diff
@@ -0,0 +1,114 @@
+# Copyright 2021 Huawei Technologies Co., Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ============================================================================
+from tinyms import layers
+from tinyms import ParameterTuple
+
+from tinyms.context import ParallelMode, get_auto_parallel_context
+from tinyms.optimizers import Adam
+from tinyms.primitives import HyperMap, GradOperation
+from tinyms.primitives import depend
+from tinyms.primitives import DType, Fill, ReduceMean, ReduceSum, Shape, Square
+from tinyms.primitives import SigmoidCrossEntropyWithLogits
+from mindspore.communication.management import get_group_size
+from mindspore.nn.wrap.grad_reducer import DistributedGradReducer
```

Comment:
Hi @kaierlong, Great job. Thanks for your submitting. I think it would be better to change `from mindspore.nn.wrap.grad_reducer import DistributedGradReducer` to `from tinyms.layers import DistributedGradReducer`.